floodviz/views.py

```diff
@@ -1,7 +1,112 @@
 from flask import request, render_template, abort
+import csv
```

Comment:
See https://www.python.org/dev/peps/pep-0008/#imports for order of imports. Within a group of imports, I like to put them in alphabetical order.